go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -432,7 +444,7 @@ func (se *Engine) reload(ctx context.Context, includeStats bool) error {
 			// We therefore don't want to query for table sizes in getTableData()
 			includeStats = false
 
-			innodbResults, err := conn.Conn.Exec(ctx, innodbTableSizesQuery, maxTableCount, false)
+			innodbResults, err := conn.Conn.Exec(ctx, innodbTableSizesQuery, maxTableCount*maxPartitionsPerTable, false)
```

Comment:
Increased this because even with our limit of 10k tables, it's possible that there are more partitions than that (and this query returns one row per partition).